### PR TITLE
StyleCI integration

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,9 +1,3 @@
-cs:
-	php-cs-fixer fix --verbose
```
Comment (OskarStark) [inline Makefile]: so there is no way to fix it locally? i nee to push and use Style CI?

**Comment by dbu** [comment]
:+1: i let you merge @Soullivaneuh once you discussed the question with @OskarStark

File: Makefile (hunk 1 above)
Comment (Soullivaneuh) [inline Makefile]: You can always use php-cs-fixer. But having a command not used anymore by travis does not make sens.

BTW, not all of the people has php-cs-fixer globally installed. Some can have a `.phar` and the `make cs` command will be useless.

Comment (OskarStark) [inline Makefile]: ok fine for me!

push the button

**Pull request merged**